assets/js/admin/discounts/index.js

```diff
@@ -17,6 +16,6 @@ jQueryReady( () => {
 	 * Show/hide conditions based on input value.
 	 */
 	products.change( function() {
-		$( '#edd-discount-product-conditions' ).toggle( products.val() );
+		$( '#edd-discount-product-conditions' ).show( products.val() );
```

Comment:
@LisaCee I believe the original implementation meant to use `.toggle()` via the `display` argument: https://api.jquery.com/toggle/#toggle-display

However `products.val()` is returning `null` or an array here and jQuery seems to consider both truthy in this case. Instead the value should be checked to not be null and that be used to determine the `display` boolean.